### PR TITLE
DAT-2836 Resolve redirects, but only if they are there :)

### PR DESCRIPTION
**Pull request by sebastianmarzjan**

### CODE REVIEW

File: extensions/wikia/CategoryExhibition/CategoryExhibitionSection.class.php [hunk 1]
```diff
@@ -46,7 +46,12 @@ public function isCategoryExhibitionEnabled() {
 			$this->categoryExhibitionEnabled = false;
 			$oTmpArticle = new Article( $this->categoryTitle );
 			if ( !is_null( $oTmpArticle ) ) {
-				$rdTitle = $oTmpArticle->getRedirectTarget();
+				if ( $this->categoryTitle->isRedirect() ) {
+					$rdTitle = $oTmpArticle->getRedirectTarget();
```
Comment (pchojnacki) [inline extensions/wikia/CategoryExhibition/CategoryExhibitionSection.class.php]: Will this work if the redirect target is also redirect ?

Comment (SebastianMarzjan) [inline extensions/wikia/CategoryExhibition/CategoryExhibitionSection.class.php]: No :-)

Comment (SebastianMarzjan) [inline extensions/wikia/CategoryExhibition/CategoryExhibitionSection.class.php]: However, it is a decision that was made to keep the behavior of this check consistent with how CategoryExhibition works

Comment (pchojnacki) [inline extensions/wikia/CategoryExhibition/CategoryExhibitionSection.class.php]: ok

**Comment by pchojnacki** [comment]
LGTM

**Comment by idradm** [comment]
DAT-2836 #QA LG

**Pull request merged**